opm/simulators/utils/readDeck.cpp

```diff
@@ -248,21 +253,19 @@ void readDeck(int rank, std::string& deckFilename, std::unique_ptr<Opm::Deck>& d
     {
         Opm::eclStateBroadcast(*eclipseState, *schedule, *summaryConfig);
     }
-    catch(const std::exception& e)
+    catch(const std::exception& broadcast_error)
     {
-        failureMessage = e.what();
+        failureMessage = broadcast_error.what();
         parseSuccess = 0;
     }
 
 #endif
 
     if (*errorGuard) { // errors encountered
         parseSuccess = 0;
+        errorGuard->clear();
     }
 
-    // print errors and warnings!
-    errorGuard->dump();
```

Comment:
Since this is no longer dumped, I assume all messages will be printed as-we-go instead of here?

Reply:
The exception part of this PR as been extracted as #2813. See [this comment](https://github.com/OPM/opm-simulators/pull/2813#issuecomment-699825031)there.